modules/hello-world/pages/sample-application.adoc

```diff
@@ -1,9 +1,180 @@
 
-include::ref:partial$beta-warning.adoc[]
+6.5@sdk:pages:partial$sample-application.adoc
+
+This version of the Travel Sample App uses the xref:6.5@server:developer-preview:preview-mode.adoc[Collections Developer Preview (DP)] 
+-- enabling Developer Preview should only be done on a development machine; 
+there is no upgrade path available from a DP-enabled Couchbase Server.
+
+== Preparation
+
+As well as the xref:start-using-sdk.adoc[Java SDK 3.0] and Couchbase Server, 
+set up as described above, you will need `git` to fetch the travel sample application code:
+
+[source,bash]
+----
+git clone git@github.com:couchbaselabs/try-cb-java.git
+----
+
+Change directory into your cloned repository, and check out the latest branch (this will most probably be enabled as the default branch).
+
+[source,bash]
+----
+git checkout 6.5.0-branch
+----
+
+Before building the Travel Sample Application, which in the case of the Java SDK utilises the Collections _Developer Preview_, you need to enable this DP feature.
+Remember, do not run this on a production server: Collections is  _Developer Preview_, and is not supported for _production_ use with Couchbase Server 6.5; 
+there is no upgrade path from a DP-enabled server.
+
+[source,bash]
+.Enable Collections Developer Preview
+----
+/opt/couchbase/bin/couchbase-cli enable-developer-preview --enable -c http://localhost:8091 -u Administrator -p password
+----
++
+[source,bash]
+----
+Developer preview cannot be disabled once it is enabled. If you enter developer preview mode you will not be able to upgrade. DO NOT USE IN PRODUCTION.
+Are you sure [y/n]: y
+----
+
+The Travel Sample Bucket needs altering to be split into collections. 
+There is a script to do this included with the Travel Sample App -- run:
+
+[source,bash]
+.Create Sample Collections
+----
+sh create-collections.sh Administrator password 127.0.0.1
+----
+
+adjusting for any changes you have made to server URL, or admin password.
+You should now have the Travel Sample Data Bucket split into collections:
+
+----
+{"uid":"1"}{"uid":"2"}{"uid":"3"}
+
+THE FINAL RESULT
+{"uid":"3","scopes":[{"name":"userData","uid":"8","collections":[{"name":"flights","uid":"9"},{"name":"users","uid":"8"}]},{"name":"_default","uid":"0","collections":[{"name":"_default","uid":"0"}]}]}
+----
+
+
+== Running the Travel Sample Application
+
+Next, edit the `storage.host` field in src/main/resources/application.properties to the one for your containerised Couchbase Server (or localhost, 127.0.0.1, if appropriate), and any other local changes -- such as password.
+From here onwards, we'll assume the defaults.
+
+And run with
+
+[source,bash]
+----
+mvn spring-boot:run
+----
+
+Most likely, you'll want to open up your preferred IDE for the `storage.host` step, 
+and stay there to build the app, rather than running Maven from the command line.
+
+After your preferred activity away from the screen for a couple of minutes of build 
```

Comment:
is it really taking minutes?

Reply:
On my network, yes, we can't all live in the 21st Century you know :-(